### PR TITLE
Changed library to fix conflicts with findbugs annotations

### PR DESCRIPTION
**Pull request by Garzas**

### CODE REVIEW

File: rx-extensions/build.gradle [hunk 1]
```diff
@@ -11,10 +11,11 @@ repositories {
 }
 
 dependencies {
-    compile "javax.annotation:javax.annotation-api:1.2"
-    compile "com.google.code.findbugs:jsr305:2.0.1"
-    compile "io.reactivex:rxjava:1.0.14"
+    compile "io.reactivex:rxjava:1.0.8"
```
Comment (jacek-marchwicki) [inline rx-extensions/build.gradle]: Why did you downgraded rxjava?

File: rx-extensions/build.gradle [hunk 2]
```diff
@@ -11,10 +11,11 @@ repositories {
...
+    compile 'com.google.guava:guava:18.0'
```
Comment (jacek-marchwicki) [inline rx-extensions/build.gradle]: We don't want dependency to guava

File: rx-android-extensions/build.gradle [hunk 3]
```diff
@@ -20,9 +20,8 @@ dependencies {
     compile 'com.android.support:support-v4:23.0.1'
     compile 'com.android.support:appcompat-v7:23.0.1'
     compile 'com.android.support:recyclerview-v7:23.0.1'
+    compile 'io.reactivex:rxandroid:0.24.0'
```
Comment (jacek-marchwicki) [inline rx-android-extensions/build.gradle]: We don't want this dependency

File: rx-extensions/build.gradle [hunk 4]
```diff
@@ -11,10 +11,11 @@ repositories {
...
     compile 'javax.inject:javax.inject:1'
+    compile 'com.google.code.findbugs:annotations:3.0.0'
```
Comment (jacek-marchwicki) [inline rx-extensions/build.gradle:17]: Wouldn't be enough to update jsr305 ? and remove jsr250  ?

File: rx-extensions/build.gradle (hunk 1 above)
Comment (Garzas) [inline rx-extensions/build.gradle]: My bad, I was looking at previous version :/

File: rx-extensions/build.gradle (hunk 2 above)
Comment (Garzas) [inline rx-extensions/build.gradle]: deleted

File: rx-android-extensions/build.gradle (hunk 3 above)
Comment (Garzas) [inline rx-android-extensions/build.gradle]: deleted

File: rx-extensions/build.gradle (hunk 4 above)
Comment (Garzas) [inline rx-extensions/build.gradle:17]: fixed

**Pull request merged**